src/shared/common/forms/confirmation-with-team-name-and-case-ref.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ConfirmationWithCaseRef extends Component {
+
+    render() {
+
+        const {
+            label,
+            caseRef,
+            nextActions,
+            teamName
+        } = this.props;
+
+        return (
+            <div>
+                <div className="govuk-panel govuk-panel--confirmation">
+                    <div className="govuk-panel__body">Case {label}: {teamName}
+                    </div>
+                </div>
+                <h2 className="govuk-heading-m">
+                    Case {caseRef} {label} {teamName}
+                </h2>
+                <div>
+                    {nextActions &&
```

Comment:
It does look like you're not using this at all as well, again is it there for a reason?

Reply:
It was part of the initial spec, to have a number of next actions, we couldn't work out a satisfactory way to do this easily within this case, I'll remove it for now